system-int-test/test/cmr/system_int_test/site/holdings_provider_test.clj

```diff
@@ -92,6 +104,19 @@
                             :beginning-date-time "2012-01-01T00:00:00.000Z"
                             :ending-date-time "2012-01-01T00:00:00.000Z"})
                          {:format :umm-json})
+        
+        _g3-online (data/ingest "PROV1"
+                         (dg/granule-with-umm-spec-collection
```

Comment:
while I know you did not create it, I think it has been decided to no longer use such short aliases, dg is not descriptive enough, data-granule would be better maybe?

Reply:
now changed to data-granule